release-scripts/do_release.py

```diff
@@ -244,9 +247,9 @@ def update_master():
     actions()
```

Comment:
We should be updating the __init__ versions for the other libraries in master.

Also, I don't see that we are updating the __init__ versions for the candidate branch either.